api/client.go

```diff
@@ -374,3 +374,23 @@ func (c *Client) Version(ctx context.Context) (string, error) {
 
 	return version.Version, nil
 }
+
+// IsLocal checks whether the client is connecting to a local server.
+func (c *Client) IsLocal() bool {
```

Comment:
I haven't thought about this extensively, but do we want to do this on the server side?

```
func isLocalhost(remoteAddr string) bool {
    host, _, err := net.SplitHostPort(remoteAddr)
    if err != nil {
        return false
    }

    return host == "127.0.0.1" || host == "::1"
}
```

Reply:
The issue I'm thinking about: Ollama running in Docker - copying files in may not work.

Reply:
I think we're trying to do it server side, just not sure how to do it yet. This is just it working with a clientside check and the POST method